mappings/net/minecraft/client/render/particle/Particle.mapping

```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/class_77 net/minecraft/client/render/particle/Particle
```

Comment:
why do they need their own package there is literally one class in that package am I not wrong?